prisma/migrations/atlas.sum

```diff
@@ -0,0 +1 @@
+h1:47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=
```

Comment:
confused by this, probably need to delete?

it doesn't match the hash in `sql/migrations/atlas.sum` and i'm pretty sure we don't usually compute this here

Reply:
Sounds good, probably something janky with me running atlas myself to try and fix things